src/https_client.cc

```diff
@@ -331,6 +331,9 @@ HTTPSResponse HTTPSClient::makeHttpRequest(
                                         Poco::Net::HTTPMessage::HTTP_1_1);
         poco_req.setKeepAlive(true);
 
+        // Require new header for all Apple token
+        poco_req.set("Referer", kTogglDesktopClientID);
```

Comment:
If we use Apple Token in Basic Authentication for `v8` or `v9` APIs, we have to add `Referer` key in the Header

Reply:
Just asking BE team. It's clear that `kTogglDesktopClientID` is only for `macOS`

I will ask some if statement to get the correct one for the future 👍 

https://toggl.slack.com/archives/CSE5U3ZUN/p1583929004134500?thread_ts=1583928914.133300&cid=CSE5U3ZUN